tests/Controller/EnMarche/MembershipControllerTest.php

```diff
@@ -101,6 +101,9 @@ public function testCannotCreateMembershipAccountWithInvalidFrenchAddress()
         $this->assertSame('Votre adresse n\'est pas reconnue. Vérifiez qu\'elle soit correcte.', $crawler->filter('#membership-address > .form__errors > li')->eq(1)->text());
     }
 
+    /**
+     * @group skip
```

Comment:
Just to be sure, this one is skipped because the feature is not implemented yet ?

Reply:
I'll skip all tests which depend on the registration.